apps/builder/app/builder/features/seo/project-settings.tsx

```diff
@@ -132,24 +128,72 @@ const ProjectSettingsContent = (props: Props) => {
           rows={5}
           autoGrow
           maxRows={10}
-          value={props.value.code ?? ""}
+          value={props.meta.code ?? ""}
           onChange={handleChange("code")}
         />
       </Grid>
-      <div />
-    </Grid>
+    </>
+  );
+};
+
+const ProjectAdvancedSettings = (props: {
+  settings: ProjectSettings;
+  onSettingsChange: (settings: ProjectSettings) => void;
+}) => {
+  const ids = useIds(["atomicStyles"]);
+
+  const handleChange =
+    <T extends keyof ProjectSettings>(name: T) =>
+    (val: ProjectSettings[T]) => {
+      props.onSettingsChange({
+        ...props.settings,
+        [name]: val,
+      });
+    };
+
+  return (
+    <>
+      <Separator />
+      <Grid gap={2} css={{ mx: theme.spacing[5], px: theme.spacing[5] }}>
+        <Label sectionTitle>Advanced Settings</Label>
+        <Text color="subtle">
+          Fine-tune your project with a range of settings, ensuring optimal site
+          functionality and performance tailored to your unique requirements.
```

Comment:
I think we can get rid of this text, because I don't see any new info user would learn from it, I don't see what the text is needed for